flink-formats/flink-sql-orc/src/main/resources/META-INF/NOTICE

```diff
@@ -6,13 +6,14 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
-- org.apache.orc:orc-core:1.5.6
-- org.apache.orc:orc-shims:1.5.6
-- org.apache.hive:hive-storage-api:2.6.0
-- io.airlift:aircompressor:0.10
-- commons-lang:commons-lang:2.6
+- org.apache.orc:orc-core:1.7.5
+- org.apache.orc:orc-shims:1.7.5
+- org.apache.hive:hive-storage-api:2.8.1
+- io.airlift:aircompressor:0.21
+- org.apache.commons:commons-lang3:3.3.2
 
 This project bundles the following dependencies under the BSD license.
 See bundled license files for details.
 
 - com.google.protobuf:protobuf-java:2.5.0
+- org.threeten:threeten-extra:1.5.0
```

Comment:
I see only these dependencies included, thus require an entry in the NOTICE
```
mvn clean install -DskipTests -Dfast -pl flink-formats/flink-sql-orc > /tmp/mvn.log
cat /tmp/mvn.log | grep 'Including' | awk '{print $3}' | sed s/:jar// | grep -v 'org.apache.flink' | sort
com.google.protobuf:protobuf-java:2.5.0
io.airlift:aircompressor:0.21
org.apache.hive:hive-storage-api:2.8.1
org.apache.orc:orc-core:1.7.5
org.apache.orc:orc-shims:1.7.5
```
@gyfora wdyt?

Reply:
Removed the redundant dependencies.